src/main/java/Pac.java

```diff
@@ -0,0 +1,229 @@
+                        }
+                        try {
+                            addEvent(inputArray[1]);
+                        } catch (ArrayIndexOutOfBoundsException e) {
+                            System.out.println(newline + "\nSorry! Command format is incorrect.\n"
+                                    + newline + "\n");
+                        }
+                        break;
+                    case DELETE:
+                        if (inputArray.length == 1) {
+                            throw new PacException("DELETE should be followed by a integer.");
+                        }
+                        try {
+                            deleteTask(Integer.parseInt(inputArray[1]) - 1);
+                        } catch (NumberFormatException e) {
+                            System.out.println(newline + "\nSorry! MARK should be followed by a integer.\n"
+                                    + newline + "\n");
+                        } catch (IndexOutOfBoundsException e) {
+                            System.out.println(newline + "\nSorry! Please mention a valid task number.\n"
+                                    + newline + "\n");
+                        }
+                        break;
+                    case ERROR:
+                        throw new PacException("I don't know what this means");
+                }
+            } catch (PacException e) {
+                System.out.println(newline + "\n" + e.getMessage() + "\n" + newline + "\n");
+            }
+        }
+    }
+
+    public static void addToDo(String description) {
+        try {
+            if (description.isBlank()) {
+                throw new PacException("Please mention a task.");
+            }
+            Task task = new ToDo(description);
+            tasks.add(task);
+            System.out.println(newline + "\nadded: " + task.toString());
+            System.out.println("You have " + tasks.size() + " tasks in your list" + "\n" + newline + "\n");
+        } catch (PacException e) {
+            System.out.println(newline + "\n" + e.getMessage() + "\n" + newline + "\n");
+        }
+    }
+
+    public static void addDeadline(String input) {
+       try {
+           String[] inputArray = input.split("/");
+           inputArray[1] = inputArray[1].replaceFirst("BY ", "by ");
+           inputArray[1] = inputArray[1].replaceFirst("By ", "by ");
+           inputArray[1] = inputArray[1].replaceFirst("bY ", "by ");
+           inputArray[1] = inputArray[1].split("by ", 2)[1];
+           if (inputArray[1].isBlank()) {
+               throw new PacException("Please enter a valid date or time.");
+           }
+           Task task = new Deadline(inputArray[0], inputArray[1]);
+           tasks.add(task);
+           System.out.println(newline + "\nadded: " + task.toString());
+           System.out.println("You have " + tasks.size() + " tasks in your list" + "\n" + newline + "\n");
```

Comment:
Maybe you can add a few blank lines in this code block to make it more readable? The coding standard suggest that "Logical units within a block should be separated by one blank line." Perhaps you could add blank lines to separate the code into sections dealing with formatting, processing and output.